charts/firehose-ethereum/README.md

```
@@ -237,7 +237,7 @@ We do not recommend that you upgrade the application by overriding `image.tag`.
  | firehoseComponentDefaults.lifecycle | Lifecycle hooks | object | `{}` |
  | firehoseComponentDefaults.nodeSelector | Node selector configuration | object | `{}` |
  | firehoseComponentDefaults.podDisruptionBudget | Pod Disruption Budget configuration | object | `{"enabled":false,"metadata":{"annotations":{},"labels":{}},"spec":null}` |
- | firehoseComponentDefaults.podManagementPolicy | (StatefulSet only), scaling behavior: (OrderedReady | Parallel) | string | `"OrderedReady"` |
+ | firehoseComponentDefaults.podManagementPolicy | , scaling behavior: (OrderedReady | Parallel) | StatefulSet only | `"OrderedReady"` |
```

Markdownlint (0.37.0)

Expected: 4; Actual: 5; Too many cells, extra data will be missing
Table column count

(MD056, table-column-count)